binstar_client/utils/yaml.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from .decorators import memoize
+
+
+@memoize
```

Comment:
I think memoize is not really necessary here. We could remove the `get_yaml` function, move the import out and avoid calling the `get_yaml` on `yaml_load` and `yaml_dump` every time. Unless there's a reason to call the import every time i think that's the path i would follow.

Reply:
Fixed